### PR TITLE
Removed rhsm tasks from ros-setup playbook

### PR DESCRIPTION
**Pull request by gnurag**
* Removed rhsm tasks for configuring rhui repos
* Moved debug statements to the top
* Formatting changes

### CODE REVIEW

File: ansible-playbooks/ros_install_and_set_up.yml [hunk 1]
```diff
@@ -3,62 +3,45 @@
...
+# 6. Start and enable pmcd service
+# 7. Start and enable pmlogger service
 
 - hosts: all
   become: yes
   gather_facts: True
   tasks:
+    - debug: var=ansible_distribution
+    - debug: var=ansible_distribution_major_version
+    - debug: var=ansible_architecture
```
Comment (apuntamb) [inline ansible-playbooks/ros_install_and_set_up.yml:17]: Do we need these variables anymore, now that we've removed the tasks that use them?

Comment (gnurag) [reply ansible-playbooks/ros_install_and_set_up.yml:17]: You're right - we don't need it anymore. I was thinking of adding a banner on top saying _ROS Configurator - RHEL-8.4 x86_64_ or something, but lets remove it for now.

**Review by apuntamb — APPROVED** [review]
lgtm :+1:

**Comment by apuntamb** [comment]
Merged, thanks @gnurag!

**Comment by gnurag** [comment]
Thanks +1

**Pull request merged**